src/content/pages/index.md

```diff
@@ -84,38 +69,6 @@ tokenHolderRole:
       variant: INTERNAL_OUTLINE
   rowReverse: true
   bgColor: "#3c3c3c"
-migrationInfo:
-  title: Do you own KEEP or NU?
-  description: Keep and NuCypher merged to form the Threshold Network. Upgrade
-    your tokens to T!
-  image: /images/legacy-tokens.png
-  buttons:
-    - label: Upgrade tokens
-      url: https://dashboard.threshold.network/upgrade
-      variant: EXTERNAL_OUTLINE
-  bgColor: "#141414"
-  rowReverse: false
-tacoRole:
-  title: Build uncensorable e2ee into your dapp
-  description: TACo is the only end-to-end encryption plugin that is end-to-end decentralized
-  image: /images/taco-section-image.svg
-  buttons:
-    - label: Learn More
-      url: /build/taco
-      variant: INTERNAL_SOLID
-    - label: Quickstart
-      url: https://docs.threshold.network/app-development/threshold-access-control-tac/quickstart-testnet
```

Comment:
For at least the duration of the transition period, let's retain a reduced TACo section which links to the new landing page & docs. This section can be moved to the foot of the homepage if necessary to avoid distracting inbound tBTC users. To minimize signals of disruption, let's reuse the same language+image and just edit the buttons+links:

 title: Build uncensorable e2ee into your dapp
  description: <a href="https://taco.build">TACo</a> is the only end-to-end encryption plugin that is end-to-end decentralized
  image: /images/taco-section-image.svg
  buttons:
    - label: Taco.build
      url: https://taco.build
    - label: Docs
      url: https://docs.taco.build
      
(note GH doesn't allow suggestions on deleted lines)